services/QuillLMS/spec/models/prompt_health_spec.rb

```diff
@@ -32,4 +32,22 @@
   it { should validate_inclusion_of(:percent_reached_optimal).in_range(0..100)}
   it { should validate_inclusion_of(:difficulty).in_range(0..5)}
 
+  context '#serializable_hash' do
+    it 'serialize into the expected shape' do
+      prompt_health = create(:prompt_health)
+      assert_equal prompt_health.serializable_hash, {
```

Comment:
I would recommend using the rspec template of
`expect(____).to eq ____` over `assert_equals` since this communicates what is "actual" vs. "expected".  Here's a good [post](https://gist.github.com/myronmarston/5889730)